AppSettingsManagerBff/AppSettingsManagerBff.Infrastructure/ApiRepositories/HttpUserRepository.cs

```diff
@@ -1,18 +1,13 @@
-using System.Text.Json;
 using AppSettingsManagerBff.Domain.ApiRepositories;
-using AppSettingsManagerBff.Model.Api;
+using AppSettingsManagerBff.Model;
+using AppSettingsManagerBff.Model.Requests;
 
 namespace AppSettingsManagerBff.Infrastructure.ApiRepositories;
 
 public class HttpUserRepository : IUserRepository
```

Comment:
Adding on to what I said before in the IUserRepository file. This class definitely won't work since the contents of ISettingRepository are copied into IUserRepository